src/components/dashboard/HeaderDasboard.tsx

```
@@ -106,7 +106,10 @@ export default function HeaderDasboard() {
         <div className="flex items-center justify-between px-6 py-3">
           {/* Mobile menu */}
           <div className="flex items-center">
-            <button onClick={() => setSidebarOpen(true)} className="text-orange-700 mr-4 md:hidden">
+            <button
+              onClick={() => setSidebarOpen(true)}
+              className="text-orange-700 dark:text-orange-400 mr-4 md:hidden"
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)